examples/garrawarla/configs/repos.yaml

```diff
@@ -0,0 +1,8 @@
+repos:
+# this list is interpreted as a priority list, 
+# in that items higher up in the list are inspected first
+# and take precedence
+# 
+# Spack user (Pawsey staff) populates this one with fixed recipes for Setonix
+# that are not (yet) in the Spack public repo
+- /astro/pawsey0001/maali/pawsey-spack-config/setonix/repo_setonix
```

Comment:
maybe rename `maali` to `$USER`, as MarkO suggested they might just rename the `maali` user to `spack` everywhere